analyzers.yaml

```diff
@@ -66,3 +66,11 @@ spec:
               message: "Not all nodes are online."
           - pass:
               message: "All nodes are online."
+    - clusterPodStatuses:
+        name: contour pods unhealthy
+        namespaces:
+          - projectcontour
+        outcomes:
+          - fail:
+              when: "!= Healthy" # Catch all unhealthy pods. A pod is considered healthy if it has a status of Completed, or Running and all of its containers are ready.
+              message: Pod {{ .Namespace }}/{{ .Name }} is unhealthy with a status of {{ .Status.Reason }}.
```

Comment:
It would be better if we could not just indicate what the issue is but point the user to what they can do about it. Is there documentation from community, upstream, or elsewhere that we can add here? Or can we create said documentation and point to that?

Reply:
there's IME two issues here - one is "envoy pod has died for unknown reasons", and will be fixed by restarting the pods
the other is that the certificates have expired and need to be renewed

Reply:
Sure, considering this could happen for different reasons we probably can't prescribe the solution. Maybe instead we tell them where to look for more information? I presume check the logs or the previous logs for the failing container?

If we assume the user isn't a K8s expert, just telling them where to find logs for the failing item would probably be useful.

Reply:
this comment thread is outdated, current message is as follows:
```
message: A Contour pod, {{ .Name }}, is unhealthy with a status of {{ .Status.Reason }}. Restarting the pod may fix the issue.
```